src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,90 @@
+package parser;
+
+import java.util.Arrays;
+
+import exception.WillyException;
+import task.TaskList;
+
+public class Parser {
```

Comment:
Consider adding Javadocs for this class.

Reply:
Hello thanks for feedback. I forgot to put javadocs and will add them in the following few commits. 

Can I check if it impacts the current grading assessment for my past work because I forgot to include it?

I will definitely include in the next few commits.